src/lib/media-query/breakpoints/break-point-registry.ts

```diff
@@ -30,6 +30,20 @@ export class BreakPointRegistry {
   }
 
   /**
+   * Accessor to sorted list used for
```

Comment:
So I do not mind these comments. It is my hope however that reviewers will focus on content not format.